py-utils/src/utils/msg_bus/.gitignore

```diff
@@ -0,0 +1,4 @@
+*.pyc
```

Comment:
Please rename old message_bus to "msg_bus" and move this code to "message_bus" folder.

Reply:
Updated and now old code can be found in msg_bus new code in message_bus